charts/camunda-platform-8.6/templates/camunda/_helpers.tpl

```diff
@@ -561,7 +561,11 @@ Release templates.
   {{- $baseURL := printf "%s://%s" $proto .Values.global.ingress.host }}
 
   {{- if .Values.console.enabled }}
-  {{- $baseURLInternal := printf "http://%s.%s:%v" (include "console.fullname" .) .Release.Namespace .Values.console.service.managementPort }}
+  {{- $proto := "http" -}}
+  {{- if eq (lower .Values.console.readinessProbe.scheme) "https" -}}
```

Comment:
There is no need to make if statement, just use the scheme directly.

```
{{-  $proto := (lower .Values.console.readinessProbe.scheme) -}}
```

Reply:
done